### PR TITLE
[2.11.x] DDF-3273 Disable Java Security Manager & update hardening guide re: SSH

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Turns off security manager.

#### Who is reviewing it? 
@tbatie @paouelle 

(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard
@lessarderic
@ricklarsen - Documentation
@stustison

#### How should this be tested? (List steps with links to updated documentation)
Full CI build. Full regression pass.

#### What are the relevant tickets?
[DDF-3273](https://codice.atlassian.net/browse/DDF-3273)
[DDF-3297](https://codice.atlassian.net/browse/DDF-3297)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by coyotesqrl** [comment]
build now

File: distribution/ddf-common/src/main/resources/common-bin.xml [hunk 1]
```diff
@@ -58,8 +58,8 @@
 
                 <!--
                 Skipping customized libraries.
-                -->
                 <exclude>**/org/apache/aries/blueprint/org.apache.aries.blueprint.core/1.8.2/org.apache.aries.blueprint.core-1.8.2.jar</exclude>
+                -->
```
Comment (millerw8) [inline distribution/ddf-common/src/main/resources/common-bin.xml]: Do these comments need to be removed to turn on the Security Manager?  If so, we should probably explain that in the remarks above

**Comment by paouelle** [comment]
Should we also update the doc around export/import to indicate SSH has to be turned back on temporarily? Also updates to the HA docs for the same.

**Comment by paouelle** [comment]
❗️ If you do a backport of this PR to master, make sure to also comment out the equivalent lines in the setenv-xxx-wrapper.conf files. Looks like since we haven't merged the OOM PR to 2.11.x, I will have to go comment it out there before merging.

File: distribution/ddf-common/src/main/resources/common-bin.xml (hunk 1 above)
Comment (coyotesqrl) [reply distribution/ddf-common/src/main/resources/common-bin.xml]: Honestly, I left them in with the expectation they'll be removed in the next week or two. Iff someone wanted to build a DDF with our security manager turned on they'd also have to include this hacked file; however, they'd also have to hack more files. This is here more as a hint on how to drop in the hacked files.

Usually, I'd rely on source control for things like this but in this one case I thought it made sense to keep the hint around for a little while.

**Comment by paouelle** [comment]
❗️ Need to update the doc for the service wrapper since it requires access to the shell command to install it.

File: distribution/ddf-common/src/main/resources/common-bin.xml (hunk 1 above)
Comment (lessarderic) [reply distribution/ddf-common/src/main/resources/common-bin.xml]: Should we create a ticket to get this removed and add a TODO here?

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java [hunk 2]
```diff
@@ -607,26 +607,7 @@ private static Integer findPortNumber(Integer portToTry) {
...
-    return options(
-        HomeAwareVmOption.homeAwareVmOption(
-            "-Dprograde.generated.policy={karaf.home}/generated.policy"),
-        vmOption("-Dprograde.use.own.policy=true"),
-        vmOption("-Djava.security.manager=net.sourceforge.prograde.sm.PolicyFileGeneratorJSM"));
-  }
-
-  private Option[] standardSecurityManager() {
-    return options(vmOption("-Djava.security.manager=net.sourceforge.prograde.sm.ProGradeJSM"));
+        HomeAwareVmOption.homeAwareVmOption("-Dddf.home={karaf.home}"));
```
Comment (coyotesqrl) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: I've intentionally left this param in. I think it better mimics our runtime to set `ddf.home` prior to starting the JVM.

**Comment by coyotesqrl** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

**Pull request merged**